### DISCUSSION TITLE
Product update with purchase code token

### DISCUSSION THREAD

**Discussion by ali-kianii** [codepotent/Update-Manager#62] [Ideas]
How to allow people who have purchased this product on WooCommerce to update their product?

**Comment by johnalarcon** [comment]
Woo Commerce is a WordPress-only plugin and it doesn't work with ClassicPress. You can use _Easy Digital Downloads_ or _Woo Commerce_ to push updates for your premium products.

**Comment by ali-kianii** [comment]
I have both templates and plugins for sale on my site
Any plugin or template with any purchase code must be enabled in only one domain
And only the same domain should be able to update
I really need this and I have no one to help me
Please help me, this is my wish for many years

**Reply by johnalarcon** [reply]
The Update Manager plugin **does not work** with Woo Commerce and there are no plans to make it compatible. You will need to use [this extension](https://woocommerce.com/products/woocommerce-api-manager/) to handle your licensing and updates.
